lib/combinat.gd

```diff
@@ -1154,17 +1154,19 @@ DeclareGlobalFunction( "IteratorOfPartitions" );
 
 #############################################################################
 ##
-#F  IteratorOfPartitionsSet( <set> )
+#F  IteratorOfPartitionsSet( <set> [, <k> [ <flag> ] ] )
```

Comment:
The name `flag` is IMHO not very nice. Then again, I have no better / good alternative (and it's better to use a bland name that forces one to read the docs over something potentially ambiguous). So I'll accept it, but if anybody can think of something better, don't hesitate to suggest it :-)

Reply:
It's not the most creative suggestion, but what about '[or]smaller[Size]'?